src/main/kotlin/com/cognifide/gradle/aem/api/AemConfig.kt

```diff
@@ -97,6 +97,13 @@ class AemConfig(
     @Input
     var deployDistributed: Boolean = propParser.boolean("aem.deploy.distributed", false)
 
+    /**
+     * Determines if connection to untrusted (e.g. self-signed) SSL certificates should be allowed.
+     * By default allows all ssl connections.
+     */
+    @Input
+    var deployTrustingAllSSLCertificates: Boolean = propParser.boolean("aem.deploy.trustingAllSSLCertificates", true)
```

Comment:
the parameter is about connection so I am opting for preserving `deployConnection` prefix (in the future I am planning even to group parameters into subsections so that keeping it grouped have sense)